build/RepositoryBuild.targets

```diff
@@ -23,13 +24,22 @@
   </Target>
 
   <Target Name="_BuildRepositories" DependsOnTargets="GetRepoBatches" Condition=" @(RepositoryBuildOrder->Count()) != 0 ">
+    <RemoveDir Directories="$(_SignRequestIntermediateOutputPath)" />
```

Comment:
I assume you're just ensuring clean directories here?

Reply:
yes.